packages/web/src/components/spa/JobsSection/JobsSection.jsx

```diff
@@ -1,20 +1,84 @@
+import React from 'react';
 import { NavBar } from '../Navigation/NavBar';
-import './jobs_section.css';
+import axios from 'axios';
+import { useState } from 'react';
 import { Anchor, Props } from '../Navigation/NavFactory';
+import { useEffect } from 'react';
+import { url } from '../../../Global';
+import { JobsCard } from './JobsCard';
+import { Footer } from '../Footer/Footer';
+import './jobs_section.css';
 
 const anchorList = [
     new Anchor('About', '/#about'),
-    new Anchor('Events', '/#events'),
-    new Anchor('Articles', '/#articles'),
     new Anchor('Team', '/#team'),
     new Anchor('Jobs', '/jobs')
 ];
 
 export const JobsSection = () => {
-    return (
-        <div className="jobs-container">
-            <NavBar props={new Props(anchorList, true, 'transparent', false)} />
-            <h1>WORKING</h1>
-        </div>
-    );
+    const [bodyHeight, setBodyHeight] = useState(0);
+    let currentBodyHeight = document.documentElement.scrollHeight;
+
+    if (currentBodyHeight !== bodyHeight) {
+        setBodyHeight(currentBodyHeight);
+    }
+    console.log();
+    const [jobs, setJobs] = useState([{}]);
+
+    const getJobs = () => {
+        axios({
+            method: 'get',
+            url: url + '/api/job'
+        })
+            .then((res) => {
+                setJobs(res.data.data.data);
+            })
+            // eslint-disable-next-line no-unused-vars
+            .catch((err) => {});
+    };
+    useEffect(() => {
+        getJobs();
+    }, []);
+    if (jobs) {
+        return (
+            <div className="jobs-page">
+                <NavBar props={new Props(anchorList, true)} />
+
+                <div
+                    className="card-section"
+                    style={{
+                        minHeight: currentBodyHeight - 0.2 * currentBodyHeight
+                    }}
+                >
+                    <div className="jobs-card">
```

Comment:
Last thing, can you change this to jobs-section-body or something like this. The div name is not correctly portraying what it holds.